gms/gossiper.cc

```diff
@@ -2311,6 +2311,9 @@ future<> gossiper::wait_alive(std::vector<gms::inet_address> nodes, std::chrono:
 }
 
 future<> gossiper::wait_for_live_nodes_to_show_up(size_t n) {
+    if (this_shard_id() != 0) {
+        on_internal_error(logger, "Must be called on shard 0");
```

Comment:
It's  better to have unique assertion-failure messages that you can grep without looking at full backtraces. For example, this message can be something like "gossiper::wait_for_live_nodes_to_show_up must be called on shard 0".

Reply:
okay